requirements.txt

```diff
@@ -14,3 +14,6 @@ numpy>=1.9.0,<1.12.0; python_version < '2.7'
 numpy>=1.9.0; python_version >= '2.7'
 pyopenssl>=0.15
 docopt
+
+# Use psutil for scheduler TEST_LOG_MONITORING
+psutil
```

Comment:
If it's for testing, I think psutil not be in the requirements for the production instance no?

Reply:
Yes it should ... on production you will need to check how much your scheduler is consuming on system resources and then you need it